src/cli/cmd/init/prompt.rs

```diff
@@ -1,27 +1,32 @@
-use std::fmt::Display;
+use std::{fmt::Display, process::exit};
 
 use inquire::{Confirm, MultiSelect, Select, Text};
 
 pub(super) struct Prompt;
 
 impl Prompt {
     pub(super) fn bool(msg: &str) -> bool {
-        Confirm::new(msg).prompt().expect("TODO")
+        match Confirm::new(msg).prompt() {
+            Ok(b) => b,
+            Err(_) => exit(0),
```

Comment:
I think this should be `exit(1)` instead. If the user hits `^C` in most other programs, that's an error, so it should also be one here.